### PR TITLE
Various fixes around handling maven output

### PR DESCRIPTION
**Pull request by ppalucha**

### CODE REVIEW

**Review by ksobolew — APPROVED** [review]
🤞

File: wrapper.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```
Comment (losipiuk) [inline wrapper.sh]: nit: call the file `prepare_maven_wrapper.sh`

File: common.sh [hunk 2]
```diff
@@ -90,12 +90,12 @@ function mavenMetadataPath() {
   echo $metadataPath
 }
 
+# shellcheck disable=SC2016
```
Comment (losipiuk) [inline common.sh]: why is that needed? should no longer be if you get rid of `'${project.version}'`

**Review by losipiuk — APPROVED** [review]

**Review by losipiuk — APPROVED** [review]

**Comment by ksobolew** [comment]
Any chance in merging it soon? I'd like to release `starburst-atlas` before I go on vacation ;)

**Comment by ppalucha** [comment]
> Any chance in merging it soon? I'd like to release `starburst-atlas` before I go on vacation ;)

But did you test it?

**Comment by losipiuk** [comment]
>> Any chance in merging it soon? I'd like to release starburst-atlas before I go on vacation ;)

> But did you test it?

Sure not - I kinda assumed we can pin to specific version and this will not break anything. Sorry for the ignorance

**Comment by ksobolew** [comment]
Does it need to be pulled to `release-flows` or can I just use it :)

**Comment by losipiuk** [comment]
IIUC now you can use it using `@0686ec2f66a0c024c11da7d69f213dd02d2be56b` as a version. 
Not sure if you could while it was still PR.

**Comment by ksobolew** [comment]
It worked :)

https://github.com/starburstdata/starburst-release-flows/actions/runs/4818852351

**Comment by ksobolew** [comment]
But I executed it from this branch: starburstdata/starburst-release-flows#481

**Comment by losipiuk** [comment]
Cool. I will tag it as 1.1.8

**Comment by losipiuk** [comment]
>Cool. I will tag it as 1.1.8

It got tagged automatically (or someone (@ppalucha ? ) did it)

**Comment by ppalucha** [comment]
> It got tagged automatically (or someone (@ppalucha ? ) did it)

I tagged it.
